tests/runtest.cmd

```diff
@@ -132,6 +136,68 @@ set "__TestWorkingDir=%__RootBinDir%\tests\%__BuildOS%.%__BuildArch%.%__BuildTyp
 if not defined XunitTestBinBase       set  XunitTestBinBase=%__TestWorkingDir%
 if not defined XunitTestReportDirBase set  XunitTestReportDirBase=%XunitTestBinBase%\Reports\
 
+REM At this point in the script there will be a divergence in how the tests are run
+REM For official builds we will continue to run tests using the un-unified scripting
+REM which relies on msbuild and calls runtest.proj directly. For all other scenarios
+REM runtest.py will handle setup and calls runtest.proj.
```

Comment:
nit: grammar is weird. should it say "runtest.py will handle setup and will then call runtest.proj"